core/runtimes/FiberRuntime/index.js

```diff
@@ -139,28 +139,17 @@ FiberRuntime.prototype._hasDeadLock = function () {
 FiberRuntime.prototype._genInitialQueue = function () {
   var self = this;
   var queue = [];
-  var keys = Object.keys(self._list);
```

Comment:
Fantastic!  Ummm, what's a reducer?!

On Wed, Jun 1, 2016 at 1:43 PM, Dan Rumney <notifications@github.com> wrote:

> In core/runtimes/FiberRuntime/index.js
> <https://github.com/jpaulm/jsfbp/pull/60#discussion_r65408400>:
>
> > @@ -139,28 +139,17 @@ FiberRuntime.prototype._hasDeadLock = function () {
> >  FiberRuntime.prototype._genInitialQueue = function () {
> >    var self = this;
> >    var queue = [];
> > -  var keys = Object.keys(self._list);
>
> I simplified this using a reducer
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jpaulm/jsfbp/pull/60/files/09d4df7b8b09c62b6b0f9b157bb2e55cc55fc74d#r65408400>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AATGJyvbqdV3-rqqODeKmg9WmykSqiMYks5qHcTSgaJpZM4IrzQO>
> .
>